addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -90,18 +96,20 @@ export class Chatbot extends Record {
             return;
         }
         if (this.steps.at(-1)?.eq(this.currentStep)) {
-            const nextStep = await rpc("/chatbot/step/trigger", {
+            const storeData = await rpc("/chatbot/step/trigger", {
                 channel_id: this.thread.id,
                 chatbot_script_id: this.script.id,
             });
-            if (!nextStep) {
+            if (!storeData) {
```

Comment:
This is not a good pattern with store data, though ok for now.
What if some day we want to return something, but still consider `isLast` to be true?
Ideally data should always be inserted.